c#/Mandoline.Api.Examples/Tests/UserTests.cs

```diff
@@ -4,44 +4,43 @@
 // root for full license information.
 // </copyright>
 
-namespace Tests
-{
-    using System;
-    using System.Threading.Tasks;
-    using Core;
-    using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Threading.Tasks;
+using Core;
+using Core.Client;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Tests;
 
-    [TestClass]
-    public class UserTests
+[TestClass]
+public class UserTests
+{
+    // gets current user
+    // expected: returned API key should match the one in settings
+    [TestMethod]
+    public async Task UserGetTest()
     {
-        // gets current user
-        // expected: returned API key should match the one in settings
-        [TestMethod]
-        public async Task UserGetTest()
-        {
-            var output = new TestOutput();
-            await User.RunGetUserAsync(output);
-            Assert.AreEqual(AppConstants.ApiToken, output.ReturnValueStr);
-        }
+        TestOutput output = new TestOutput();
+        await User.RunGetUserAsync(output);
+        Assert.AreEqual(AppConstants.ApiToken, output.ReturnValueStr);
+    }
 
-        // returns user object corresponding to provided credentials
-        // expected: returned API key should not be blank
-        [TestMethod]
-        public async Task UserLoginTest()
-        {
-            var output = new TestOutput();
-            await User.RunLoginAsync(output, AppConstants.UserName, AppConstants.UserPassword);
-            Assert.AreNotEqual(string.Empty, output.ReturnValueStr);
-        }
+    // returns user object corresponding to provided credentials
+    // expected: returned API key should not be blank
+    [TestMethod]
+    public async Task UserLoginTest()
+    {
+        TestOutput output = new TestOutput();
```

Comment:
Also I don't know if stylecop is telling you too add types here, but the use of var is fine here as the type is obvious because of the initialization on the same line.

Reply:
I think all var changes were automated changes when upgrading to dotnet 6